scaleway/resource_lb_backend_beta_test.go

```diff
@@ -9,11 +9,13 @@ import (
 	"github.com/scaleway/scaleway-sdk-go/api/lb/v1"
 )
 
-func TestAccScalewayLbBackendBeta(t *testing.T) {
+func TestAccScalewayLbBackend_Basic(t *testing.T) {
+	tt := NewTestTools(t)
```

Comment:
Could we use some kind of test helper here?
```
func TestAccScalewayLbBackend_Basic(t *testing.T) {
    parrallelTest(t, func(tt *TestTools) resource.TestCase {
        return resource.TestCase{
		PreCheck:     func() { testAccPreCheck(t) },
		ProviderFactories: tt.ProviderFactories,
		CheckDestroy:      testAccCheckScalewayLbBackendBetaDestroy(tt),
		Steps: []resource.TestStep{...}.
	}
    })
}
```

Or something even more abstract with our own testCase structure?
```
func TestAccScalewayLbBackend_Basic(t *testing.T) {
    parrallelTest(t, testCase {
	CheckDestroy:      testAccCheckScalewayLbBackendBetaDestroy(tt),
	Steps: []resource.TestStep{...}.
    })
}
```

Reply:
I would prefer as much as possible to have tests that looks like examples from official Terraform documentation to facilitate the onboarding of tests from the community.